<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tof_powder_plot</class>
 <widget class="QWidget" name="tof_powder_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1965</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="options_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="tB_up">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>previous dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_down">
         <property name="text">
          <string>next dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_deselect">
         <property name="text">
          <string>deselect all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_raw">
         <property name="text">
          <string>Raw</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_separated">
         <property name="text">
          <string>Sep</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="lV_datalist">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="plot_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="plot_head_layout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QComboBox" name="combB_x_axis_scale">
         <item>
          <property name="text">
           <string>2Î¸</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>q</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>d</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cB_log_scale">
         <property name="text">
          <string>log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_grid">
         <property name="text">
          <string>grid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_linestyle">
         <property name="text">
          <string>linestyle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_errorbar">
         <property name="text">
          <string>errorbar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
